ansible/roles/qemu_guest/meta/main.yml

```diff
@@ -0,0 +1,16 @@
+galaxy_info:
+  role_name: qemu_guest
+  author: Elkhan Mammadli
+  description: Installs qemu-guest-agent
+  license: MIT
```

Comment:
Wouldn't you mind if I move it to the [almalinux.ci](https://github.com/AlmaLinux/ansible-almalinux.ci) Ansible namespace after merging? Or you can create a PR by yourself, as you wish.

Reply:
I think it would be better to mention rsync in the description like "Installs qemu-guest-agent and rsync".